File: frameworks/spring-boot-orm/src/main/java/com/fraiseql/entities/Comment.java
package com.fraiseql.entities;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;
import lombok.Builder;

import javax.persistence.*;
import java.time.LocalDateTime;

@Entity
@Table(name = "tb_comment", schema = "benchmark")
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Comment {

    @Id
    @Column(name = "pk_comment")
    private Integer pkComment;

    @Column(name = "id")
    private String id;

    @Column(name = "fk_post")
    private Integer fkPost;

    @Column(name = "fk_author")
    private Integer fkAuthor;

    @Column(name = "content")
    private String content;

    @Column(name = "created_at")
    private LocalDateTime createdAt;
}

File: frameworks/apollo-server/src/entities/Comment.ts
import { Entity, PrimaryColumn, Column } from 'typeorm';

@Entity({ schema: 'benchmark', name: 'tb_comment' })
export class Comment {
  @PrimaryColumn('uuid')
  id!: string;

  @Column('text')
  content!: string;

  @Column('uuid')
  post_id!: string;

  @Column('uuid')
  author_id!: string;

  @Column('timestamp')
  updated_at!: Date;
}
